.github/workflows/tests.yml

```diff
@@ -50,9 +50,23 @@ jobs:
 
       - name: Precompile assets
         run: bundle exec rake assets:precompile
-      
+
```

Comment:
praise: thanks for the cleanup!

Reply:
Don't thank me, thank the auto-whitespace removal in Atom 😄